jplag/src/main/java/de/jplag/reportingV2/reportobject/model/Cluster.java

```diff
@@ -0,0 +1,27 @@
+package de.jplag.reportingV2.reportobject.model;
+
+import java.util.List;
+
+public class Cluster {
+    private final float average_similarity;
+    private final float strength;
+    private final List<String> members;
+
+    public Cluster(float average_similarity, float strength, List<String> members) {
+        this.average_similarity = average_similarity;
```

Comment:
underscore is not aligned with Java Naming Conventions

Reply:
I left the attributes with an underscore naming in order to easily match them to the json generated file. I will look up the Jackson annotaions. I believe there was an annotation to have different names for the same attribute in java and json.